isso/js/app/isso.js

```diff
@@ -125,13 +102,12 @@ var Postbox = function(parent) {
 
         api.create($("#isso-thread").getAttribute("data-isso-id"), {
             author: author, email: email, website: website,
-            text: utils.text($(".isso-textarea", el).innerHTML),
+            text: $(".isso-textarea", el).value,
             parent: parent || null,
             title: $("#isso-thread").getAttribute("data-title") || null,
             notification: $("[name=notification]", el).checked() ? 1 : 0,
         }).then(function(comment) {
-            $(".isso-textarea", el).innerHTML = "";
-            $(".isso-textarea", el).blur();
```

Comment:
Was the focus loss (`Node.blur()` taken out intentionally? I'm guessing because we don't need to re-add the placeholder text on `blur` event, no sense in triggering it?

Reply:
Yeah, I'm using the `placeholder` attribute on the textarea. And I feel like it would already trigger the event anyways, since clicking the submit button would make the textarea lose focus. Regardless I can't tell any difference when adding/removing that line.